modules/claims_api/app/swagger/claims_api/v2/swagger.json

```diff
@@ -597,6 +597,282 @@
         }
       }
     },
+    "/veterans/{veteranId}/claims/{claimId}": {
+      "get": {
+        "summary": "Find claim by ID",
+        "tags": [
+          "Claims"
+        ],
+        "operationId": "findClaimById",
+        "security": [
+          {
+            "productionOauth": [
+              "claim.read"
+            ]
+          },
+          {
+            "sandboxOauth": [
+              "claim.read"
+            ]
+          },
+          {
+            "bearer_token": [
+
+            ]
+          }
+        ],
+        "description": "Retrieves a specific claim for a Veteran",
+        "parameters": [
+          {
+            "name": "veteranId",
+            "in": "path",
+            "required": true,
+            "description": "ID of Veteran",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "claimId",
```

Comment:
Hmmmm, I would think this should just be `id` because it looks like the controller is looking for [`params[:id]`](https://github.com/department-of-veterans-affairs/vets-api/pull/7455/files#diff-0f0dab107f9fa58ad7bfa8892ac1a5b69f069a82aac2d926c63f12e4839a0a8dR23).

Reply:
I definitely agree with the consistency angle, and `{id}` seems pretty conventional. Updated to change it to that.